ptp/components/text/sentence_indexer.py

```diff
@@ -204,8 +220,21 @@ def tensor_distributions_to_sentences(self, data_dict):
                 output_word = self.ix_to_word[token]
                 # Add index to outputs.
                 output_sample.append( output_word )
+
+            # Apply fixed padding to all sequences if requested
+            # Otherwise let torch.nn.utils.rnn.pad_sequence handle it and choose a dynamic padding
+            if self.fixed_padding > 0:
+                pad_list(output_sample, self.fixed_padding)
+
             # Add sentence to batch.
             outputs_list.append(output_sample)
 
         # Create the returned dict.
         data_dict.extend({self.key_outputs: outputs_list})
+
```

Comment:
I  guess that function should be removed (as it is already in utils/word_mappings.py